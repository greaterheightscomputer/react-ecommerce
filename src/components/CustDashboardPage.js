import React from 'react';
import CustList from './CustList';
import CustListFilters from './CustListFilters';
import 'bootstrap/dist/css/bootstrap.min.css';

const CustDashboardPage = () => (    
    <div>                
        <CustListFilters/>        
        <CustList />                       
    </div>    
);

export default CustDashboardPage;