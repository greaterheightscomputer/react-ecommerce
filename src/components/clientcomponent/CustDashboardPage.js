import React from 'react';
import ProductProvider  from './context';
import CustList from '../../components/clientcomponent/CustList';
import CustListFilters from '../../components/clientcomponent/CustListFilters';
import 'bootstrap/dist/css/bootstrap.min.css';
import CustProductModal from './CustProductModal';
import CustCart from '../cart/CustCart';

const CustDashboardPage = () => (    
    <ProductProvider>                
        <CustListFilters/>        
        <CustList />      
        <CustProductModal/> 
        {/*<CustCart/>*/}
    </ProductProvider>    
);

export default CustDashboardPage;

