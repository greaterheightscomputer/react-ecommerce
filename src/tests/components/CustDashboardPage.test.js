import React from 'react';
import { shallow } from 'enzyme';
import CustDashboardPage from '../../components/CustDashboardPage';

test('should render CustDashboardPage correctly', () => {
    const wrapper = shallow(<CustDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});