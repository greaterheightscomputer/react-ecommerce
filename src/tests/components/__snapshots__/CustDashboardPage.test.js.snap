// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CustDashboardPage correctly 1`] = `
<div>
  <Connect(CustListFilters) />
  <Connect(CustList) />
</div>
`;
